### ISSUE TITLE
Groups - Invite my colleagues

### ISSUE THREAD

**Issue by JOutram77** [gctools-outilsgc/gccollab#37]
Owners of groups are able to invite people to their groups. One option is by email address. This does not currently work.

Can this be enabled/fixed?

**Comment by ToferC** [comment]
Just tested it and it was working for me. Only invites people on platform, but choose from drop-down.

**Comment by ToferC** [comment]
Confirmed that this works on TBS computers.  You need to enter the full email address and then just stop and wait.  Seems slow.  Not great UI.  Also graphical glitch with the delete trashcan.  An upgrade to this function, possibly with autosuggest might be a good enhancement item for Connex/Collab.

![collab group invitation](https://cloud.githubusercontent.com/assets/6226762/21851910/20931826-d7df-11e6-88f3-de7b27eed57f.PNG)

**Comment by ToferC** [comment]
@JOutram77 - please close this issue if it's no longer applicable.  Thanks!

**Comment by JOutram77** [comment]
@ToferC Happy to close this issue but this invitation field does not allow group owners to invite non-registered users. 

Do we not want to enable group owners to invite users that may not be registered GCcollab users? This would support the by-invitation access system we hope to implement later this month.

**Comment by markwooff** [comment]
I believe this was added in with a GCconnex pull request here https://github.com/gctools-outilsgc/gcconnex/pull/739 which just needs to be pulled into GCcollab. An email can be sent out to non-registered users but they would still need to register to access any GCcollab content.

The by-invitation system is a separate issue which would allow non-standard email addresses to register on GCcollab. Still waiting on approval for developing that.
